### ISSUE TITLE
Unblocking access to kitchen-ec2

### ISSUE THREAD

**Issue by robcoward** [robcoward/training-101#18]
@tyler-ball I'm not sure if this message will get to you or if it will get blocked as well, or even if you can help. I'm not sure how else I'm supposed to contact someone about this even.

Somehow I appear to have been blocked on the kitchen-ec2 repository, unable to comment on or close my own issue, create a new issue, or even fork the repo. I was hoping to close https://github.com/test-kitchen/kitchen-ec2/issues/106 as no longer relevant since you just release v0.9.1

Any chance you could help please ?
Thanks
Rob

**Comment by tyler-ball** [comment]
@robcoward Super weird - I'll start looking into it.  Are you still having issues this week?  Its a public repo so I don't see any reason you would be blocked from updating issues.  Do you get a specific error?  Have you pinged github support?

**Comment by robcoward** [comment]
@tyler-ball I haven't pinged github support yet. If i try adding any comment to issues i get the error:
"Sorry! We couldn't save your comment. Please try again."
I've also had a message display across the top of the screen saying that i was blocked on the repository, though cannot recall what i was doing at the time to reproduce it.
I'm on  holiday this week, so only have access via my phone. Will get back onto this next week when I'm back at work.
Thanks for your help, and great work on the kitchen-ec2 driver too btw.
Rob

**Comment by tyler-ball** [comment]
@robcoward I don't think we've done anything to disable you, so I would email support as your first step when you get back.  Enjoy your holiday!

**Comment by robcoward** [comment]
Hi @tyler-ball,
I've finally had a response from Matt Hartley at Github support. To quote his response:

```
Sorry for the delay in replying here! At the end of March we received a request from one of the 
test-kitchen account's Owners Team members to have your account blocked from the organization. 
Any request to have your account unblocked will need to also come from an Owners Team member.

You may want to suggest to @tyler-ball that the Owners Team discuss the decision and contact us
directly if they agree to reinstate your access to the organization's projects.
```

I assume by 'Owners Team' Matt is referring to the people listed at https://github.com/orgs/test-kitchen/people ? I would like to be able to continue contributing to the test-kitchen project, and currently have a particular interest in getting windows guest support working in the kitchen-ec2 driver. Would you be able to find out why I was blocked in the first place and see if anyone objects to reinstating my access please ?

Thanks in advance.
Rob
